src/routes/api.ts

```diff
@@ -12,18 +19,64 @@ router.get('/', (req: Request, res: Response) => {
 });
 
 //Alive check
+//Status check
 router.get('/status', (req, res) => {
-  res.status(200).send("Alive");
+  res.status(GetServerStatus().statusCode).send(GetServerStatus().statusString);
 });
 
+
+router.get('/httpTest', (req,res)=>{
+  
+  res.status(200).send("Hello from the httpTest");
+})
+
+router.post('/status/set', (req, res) => {
+  let request = req.body.status;
+
+  if(SetServerStatus(request) === "OK"){
+    res.status(200).send({"message": "Server status was updated!"}); 
+  } else {
+    res.status(400).send({"message": "Something went wrong!"});
+  }
+})
+
 //Get latestDigested
 router.get("/latest", (req, res) => {
-  let result = latestDigestedPostNumber();
-  console.log(result);
+  latestDigestedPostNumber().then(resu => 
+    res.json({
+      message: resu
+    }));
   //res.status(200).send(result);
-  res.json({
-    message: result
-  })
+  
+})
+
+//FOR THE EXPRESS DOCKERFILE!!!
+//Tested and simulated using the Backend + Postman, success!
+//API ROUTE NEEDS TO BE CHANGED TO THE CORRECT ONE! PROBABLY <host>/status
+//Please notice fetch import
+router.get('/status/404', (req,res,err) => {
```

Comment:
Also own route (all of status on 1 route)

Reply:
Moved to it's own route ('/status', status.ts), but will be removed entirely from the backend, once it's been placed on the docker file.